web/src/app/admin/AdminConfig.js

```diff
@@ -83,13 +83,12 @@ export default class AdminConfig extends React.PureComponent {
       <React.Fragment>
         <Grid
           container
-          spacing={3}
+          spacing={2}
           className={this.props.classes.gridContainer}
         >
           {groups.map((groupID, index) => (
             <Grid
               key={index}
-              container
```

Comment:
This should be both a container and item because it contains a nested grid item and is an item of a parent container.

https://material-ui.com/components/grid/#nested-grid

Reply:
It looks like that first Grid container may not be needed (A grid container holding only a grid container as its child seems at face value redundant). I'll take a look and see what's happening.